src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2019-2023 Alibaba Cloud
+// Copyright (c) 2019-2023 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::{collections::HashMap, sync::Arc};
+
+use anyhow::{anyhow, Context, Result};
+use kata_sys_util::rand::RandomBytes;
+use tokio::sync::Mutex;
+
+use crate::{
+    BlockConfig, DeviceConfig, Hypervisor, KATA_BLK_DEV_TYPE, KATA_MMIO_BLK_DEV_TYPE,
+    VIRTIO_BLOCK_MMIO, VIRTIO_BLOCK_PCI,
+};
+
+use super::{
+    util::{get_host_path, get_virt_drive_name},
+    Device,
+};
+pub type ArcMutexBoxDevice = Arc<Mutex<dyn Device>>;
```

Comment:
As Box has been deleted, I think the type name should also be changed.

Reply:
fixed